module-2/assignments/activity1.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<businesscards>
+    <!--Record Details-->
+    <data>
+        <name>Rorie Demanche</name>
+        <home>+1 (710) 153-4949 (home)</home>
+        <work>+1 (503) 319-3558 (work)</work>
+        <mobile>+1 (182) 748-1667 (mobile)</mobile>

```

Comment:
```suggestion
        <phone type="home">+1 (710) 153-4949 (home)</phone>
        <phone type="work">+1 (503) 319-3558 (work)</phone>
        <phone type="mobile">+1 (182) 748-1667 (mobile)</phone>
```
this one would be better as you can have only one element reused for any kind of phone